android/app/src/flavors/gms-enabled/AndroidManifest.xml

```diff
@@ -1,7 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please note that this file wasn't compiled in at all, even before all changes with the project structure. I tested that it isn't really needed and the app works without it on the oldest API=21 supported.